spec/index.bs

```diff
@@ -2210,20 +2180,125 @@ enum GPUTextureAspect {
     ::
         Creates a {{GPUTextureView}}.
 
-    <div algorithm=GPUTexture.createView>
-        **Called on:** {{GPUTexture}} |this|.
+        <div algorithm=GPUTexture.createView>
+            **Called on:** {{GPUTexture}} |this|.
 
-        **Arguments:**
-        <pre class=argumentdef for="GPUTexture/createView(descriptor)">
-            |descriptor|: Description of the {{GPUTextureView}} to create.
-        </pre>
+            **Arguments:**
+            <pre class=argumentdef for="GPUTexture/createView(descriptor)">
+                |descriptor|: Description of the {{GPUTextureView}} to create.
+            </pre>
 
-        **Returns:** |view|, of type {{GPUTextureView}}.
+            **Returns:** |view|, of type {{GPUTextureView}}.
 
-        Issue: write definition. |this| |descriptor| |view|
-    </div>
+            1. Set |descriptor| to the result of [$resolving GPUTextureViewDescriptor defaults$] with |descriptor|.
+            1. Issue the following steps on the [=Device timeline=] of |this|:
+                <div class=device-timeline>
+                    1. If any of the following conditions are unsatisfied:
+                        <div class=validusage>
+                            - |this| is [=valid=]
```

Comment:
Is it allowed to create a texture view on a destroyed texture?

Reply:
I would very much want this to not be allowed. In our implementation the "destroy" is a stronger version of the "drop", so it assumes the user is not going to use it explicitly for anything (like view creation). It may still be referenced from other things.

Reply:
Sounds good. The PR should include that validation rule. What about creating bindgroups or referencing directly in copy commands?

Reply:
Ideally, the same for us. For all purposes, a destroyed resource doesn't exist for the user.
We can change that implementation, I suppose, if there is a strong case for supporting this.

Reply:
> The PR should include that validation rule.

I want to clarify: "`this` is valid" implies that the texture you are calling `createView()` on is not destroyed (since explicitly calling `destroy()` is one of the things that makes a resource invalid.) Is there an additional validation you are looking for related to that?

Reply:
Ah I didn't realize that invalid and destroyed were bundled at the moment. No change is needed until we address issue 2, sorry for the confusion:

>  Consider separating "invalid" from "destroyed". This would let validity be immutable, and only operations involving devices, buffers, and textures (e.g. submit, map) would check those objects' [[destroyed]] state (explicitly).

> For all purposes, a destroyed resource doesn't exist for the user. We can change that implementation, I suppose, if there is a strong case for supporting this.

Either is fine for us. We already have code in the backends to fake stuff in VkDescriptorSet creation or D3D12 descriptor heaps so validation would allow us to move the check in the frontend. However adding the requirement for copies and beginRenderPass etc seems a bit redundant given we also check at GPUQueue.submit time.

Anyways we should probably discuss this in a different place.